analysis/beacon/collector_test.go

```diff
@@ -1,3 +1,5 @@
+// +build integration
```

Comment:
This was necessary to get unit tests to run. Running `go test -tags=integration` will run integration tests, otherwise only unit tests will be run.